internal/util/util.go

```diff
@@ -183,7 +186,11 @@ func GetVolumeSnapshotContentForVolumeSnapshot(volSnap *snapshotv1api.VolumeSnap
 
 	if err != nil {
 		if err == wait.ErrWaitTimeout {
-			log.Errorf("Timed out awaiting reconciliation of volumesnapshot %s/%s", volSnap.Namespace, volSnap.Name)
+			if snapshotContent.Status == nil && snapshotContent.Status.Error != nil {
```

Comment:
Looks like the if condition check should be
`if snapshotContent.Status == nil && snapshotContent.Status.Error != nil`

Reply:
Resolved

Reply:
!= as per other message resolved